permissions/plugin-sysdig-secure.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "sysdig-secure"
+github: "jenkinsci/sysdig-secure-plugin"
+paths:
+- "org/jenkins-ci/plugins/sysdig-secure"
+developers:
+- "alfredlandrum"
```

Comment:
We will consider this account to be the sole owner of the plugin. Since the Jira issue was filed as `sysdig`, it might make sense to add that as well, in case you change jobs.

Reply:
Thank you, that does make sense. I've added that account, and we've logged in with that account to Artifactory.